index.bs

```diff
@@ -1672,6 +1672,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 :   <code><dfn for="assertionCreationData">authenticatorDataResult</dfn></code>
                 ::  whose value is the bytes of the [=authenticator data=] returned by the [=authenticator=].
 
+                      Note: Web Browsers shall tolerate assertion formats that differ from the one described 
```

Comment:
If &ldquo;shall&rdquo; is intended to be binding, should it be &ldquo;SHALL&rdquo;?

Reply:
I am happy to make that change (plus remove the "note")